src/atoms/Img/index.js

```diff
@@ -23,14 +23,61 @@ const createName = (str) => {
   return filename ? filename[0].replace(/(\W+|_)/g, ' ').trim() : 'PolicyGenius';
 };
 
+const isPicture = (props) => {
```

Comment:
what was the linter error that led to this refactor?

Reply:
`complexity` - I had to refactor some of the code out of the `Img` function because it was too complex

Reply:
file reverted back to master and complexity ignored